inc_internal/utils.h

```diff
@@ -108,11 +107,21 @@ if (!uv_is_closing((uv_handle_t*)(h))) uv_close((uv_handle_t*)(h), (uv_close_cb)
 }}while(0)
 
 
-
 void ziti_alloc_cb(uv_handle_t *handle, size_t suggested_size, uv_buf_t *buf);
 
 int get_url_data(const char *url, struct http_parser_url *parser, int uf, char *out, size_t maxout);
 
+/**
+ * Split string based on delimiters.
+ * strings are appended to the provided list. Caller is responsible to freeing resulting strings -
+ * possibly via `model_list_clear(result, free)`
+ * @param str
+ * @param delim
+ * @param result
+ * @return number of tokens
+ */
+extern size_t str_split(const char *str, const char *delim, model_list *result);
```

Comment:
Whay extern? Risk of name conflict/linker error in user app?

Reply:
`extern` has no effect on visibility